cassandra/conf.yaml.example

```diff
@@ -157,170 +70,378 @@ init_config:
-            metric_type: counter
-            alias: jmx.gc.major_collection_count
+              metric_type: counter
+              alias: jmx.gc.major_collection_count
           CollectionTime:
-            metric_type: counter
-            alias: jmx.gc.major_collection_time
+              metric_type: counter
+              alias: jmx.gc.major_collection_time
     - include:
         domain: java.lang
         type: GarbageCollector
         name: ConcurrentMarkSweep
         attribute:
           CollectionCount:
-            metric_type: counter
-            alias: jmx.gc.major_collection_count
+              metric_type: counter
+              alias: jmx.gc.major_collection_count
           CollectionTime:
-            metric_type: counter
-            alias: jmx.gc.major_collection_time
+              metric_type: counter
+              alias: jmx.gc.major_collection_time
     - include:
         domain: java.lang
         type: GarbageCollector
         name: G1 Mixed Generation
         attribute:
           CollectionCount:
-            metric_type: counter
-            alias: jmx.gc.major_collection_count
+              metric_type: counter
+              alias: jmx.gc.major_collection_count
           CollectionTime:
-            metric_type: counter
-            alias: jmx.gc.major_collection_time
+              metric_type: counter
+              alias: jmx.gc.major_collection_time
+     # Deprecated metrics for pre Cassandra 3.0 versions compatibility.
+     # If you are using cassandra 2, the metrics below will be used,
+     # otherwise they will be ignored.
+    - include:
+        domain: org.apache.cassandra.metrics
+        type: ColumnFamily
+        bean_regex:
+          - .*keyspace=.*
+        name:
+          - ReadLatency
+          - WriteLatency
+        attribute:
+          - 75thPercentile
+          - 95thPercentile
+          - 99thPercentile
+          - OneMinuteRate
+      exclude:
+        keyspace:
+          - system
+          - system_auth
+          - system_distributed
+          - system_schema
+          - system_traces
+    - include:
+        domain: org.apache.cassandra.metrics
+        type: ColumnFamily
+        bean_regex:
+          - .*keyspace=.*
+        name:
+          - RangeLatency
+          - CasPrepareLatency
+          - CasProposeLatency
+          - CasCommitLatency
+          - ViewLockAcquireTime
+          - ViewReadTime
+        attribute:
+          - 75thPercentile
+          - 95thPercentile
+          - OneMinuteRate
+      exclude:
+        keyspace:
+          - system
+          - system_auth
+          - system_distributed
+          - system_schema
+          - system_traces
+    - include:
+        domain: org.apache.cassandra.metrics
+        type: ColumnFamily
+        bean_regex:
+          - .*keyspace=.*
+        name:
+          - SSTablesPerReadHistogram
+          - TombstoneScannedHistogram
+          - WaitingOnFreeMemtableSpace
+      attribute:
```

Comment:
The indentation here is wrong, which is why you get more histogram parts than specified for these means.

Reply:
Wow I really messed up with indentation. I fixed it for both the 'table' and 'columnfamily' metrics. Thanks!